pandas/_libs/lib.pyx

```diff
@@ -2845,15 +2845,17 @@ no_default = _NoDefault.no_default  # Sentinel indicating the default value.
 NoDefault = Literal[_NoDefault.no_default]
 
 
+@cython.boundscheck(False)
+@cython.wraparound(False)
 def map_infer_mask(
-        ndarray[object] arr,
-        object f,
-        const uint8_t[:] mask,
-        *,
-        bint convert=True,
-        object na_value=no_default,
-        cnp.dtype dtype=np.dtype(object)
-) -> "ArrayLike":
+    ndarray arr,
+    object f,
+    const uint8_t[:] mask,
+    *,
+    bint convert=True,
+    object na_value=no_default,
+    object dtype=np.dtype(object)
+) -> None:
```

Comment:
returning None looks inaccurate.  am i reading this wrong?

Reply:
sorry i mixed up the annotations from the old ``_map_infer_mask``